SimplifiedTests/NYPLAxisTests/TerminationListenerMock.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TerminationListenerMock.swift
+//  OpenEbooksTests
+//
+//  Created by Raman Singh on 2021-05-18.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+@testable import SimplyE
+
+class TerminationListenerMock: NYPLAxisItemDownloadTerminationListening {
+  
+  var willTerminate: (() -> Void)?
```

Comment:
can this be renamed to `didTerminate`, since it's called by downloaderDidTerminate? or the other way around, rename the function to `downloaderWillTerminate` -- not sure what's possible.

Reply:
didTerminate makes more sense. Done